opm/models/discretization/common/fvbasefdlocallinearizer.hh

```diff
@@ -157,26 +165,11 @@ private:
 
     using LocalEvalBlockVector = typename LocalResidual::LocalEvalBlockVector;
 
-#if __GNUC__ == 4 && __GNUC_MINOR__ <= 6
 public:
-    // make older GCCs happy by providing a public copy constructor (this is necessary
-    // for their implementation of std::vector, although the method is never called...)
-    FvBaseFdLocalLinearizer(const FvBaseFdLocalLinearizer&)
-        : internalElemContext_(0)
-    {}
-
-#else
-    // copying local residual objects around is a very bad idea, so we explicitly prevent
-    // it...
-    FvBaseFdLocalLinearizer(const FvBaseFdLocalLinearizer&) = delete;
-#endif
-public:
-    FvBaseFdLocalLinearizer()
-        : internalElemContext_(0)
-    { }
+    FvBaseFdLocalLinearizer() = default;
 
-    ~FvBaseFdLocalLinearizer()
-    { delete internalElemContext_; }
+    // copying local residual objects around is a very bad idea
```

Comment:
I suppose this comment (now) could be something along the lines of
```C++
// Disable copying for efficiency reasons
```
since, at least in principle, we could implement an actual copy constructor with value semantics.